spec/helpers/welcome_helper_spec.rb

```diff
@@ -89,7 +89,7 @@
 
     context "when stored in the mailinglist session store" do
       context "when the subject name is a common noun" do
-        %i[art_and_design biology chemistry general_science languages_other maths physics_with_maths physics].each do |subject_key|
+        %i[art_and_design biology chemistry languages_other maths physics].each do |subject_key|
```

Comment:
@plisovin Do we need to add `science` in here now that we've added it back in as a subject?